assets/.browserslistrc

```diff
@@ -0,0 +1,10 @@
+# DUP
+ChromeAndroid 61
+# Sectional
+Firefox 126
+# Pre-Fare
+Chrome 126
+# Bus Shelter
+Firefox 126
+# Bus E-Ink
+ChromeAndroid 83
```

Comment:
From our digging in the previous [task](https://app.asana.com/0/1185117109217413/1207719326862933) I thought this was going to be `Chrome 45`, based on GDS's version of qt-webkit.

Reply:
I must have been looking at an outdated list I put together. Sorry about that.